bucket/lively.json

```
@@ -4,8 +4,7 @@
     "description": "A free and open-source software that allows users to set animated desktop wallpapers and screensavers. ",
     "license": "GPL-3.0",
     "suggest": {
-        ".NET Desktop Runtime": "extras/windowsdesktop-runtime",
-        "VCRedist": "extras/vcredist"
+        ".NET Desktop Runtime": "extras/windowsdesktop-runtime"

```

```suggestion
        ".NET Desktop Runtime": "extras/windowsdesktop-runtime",
        "vcredist": "extras/vcredist2022"
```

https://github.com/rocksdanister/lively/blob/cdcfc3009e988022f440886a856eb24d2f3e4014/src/installer/Script.iss#L97